renovate.json

```diff
@@ -10,6 +10,10 @@
     {
       "matchPackagePatterns": ["^@?xstate"],
       "groupName": "xstate"
+    },
+    {
+      "matchPackageNames": ["react-monaco-editor"],
+      "allowedVersions": "<=0.41.2"
```

Comment:
shouldn't we then do the same for `@patternfly/react-code-editor` ?

Reply:
Not necessarily. If new releases of `@patternfly/react-code-editor` will still require the same version of `react-monaco-editor`, then we can still proceed with updating it. 

This is likely because they are requesting a quite old version of `react-monaco-editor`, so my bet is they will not change this peer dep requirement often.

Reply:
ok, thank you